docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+    - Platform API for Bluetooth mesh network features has been added.
+    - Open source bluez module has been upgraded to support Bluetooth mesh network.
+    - Tizen Bluetooth framework has been extended to control bluez’s mesh daemon.
+  - Tizen devices will be able to do Bluetooth mesh network provisioning and control devices like smart blubs, which support Bluetooth mesh network standard.
+- Wi-Fi multiple interface support
+  - Network modules have been extended to support devices with multiple Wi-Fi network interfaces.
+    - connman and net-config have been extended to identify a specific Wi-Fi network interface for its operations like scan and connect.
+    - Platform API for specifying Wi-Fi network interface for each operation has been added for connection and wifi-manager API.
+  - Public connection and wifi-manager API are backward compatible by introducing a concept of default Wi-Fi network interface.
+- UWB (Ultra wideband) ranging
+  - Platform API for UWB has been added.
+  - UWB manager daemon has been added.
+  - It has been tested with Decawave’s dwm1001 module on RPI4.
+  - Devices with UWB modules will be able to measure distance between them.
+
+
+### Security
+
+#### New and changed features
+
+- Security monitor
+  - Query-based security API has been added.
+  - Type-safe query builder has been added.
+  - Support for Device Policy Management has been added.
+- Privileges
+  - New privilege has been added.
+    - network.route
+
+
+### Service framework
+
+#### New and changed features
+
+- Location Framework
+  - Support for getting velocity accuracy has been added.
+  - Support for getting location bounds to check on edge with tolerance has been added.
+    - Support for checking on edge with tolerance for polygon has been added.
+    - Support for checking on edge with tolerance for circle has been added.
+    - Support for checking on edge with tolerance for rectangle has been added.
+- Battery-Monitor Framework
+  - Support for resetting battery dump data has been added.
+  - Support for dumping application id with event of battery dump has been added.
+- Context Framework
+  - Motion API has been deprecated.
+
+#### Fixes
+
+- Context Framework
+  - Fix for potential security vulnerability has been added.
+- Location Framework
+  - Precision improvement in getting distance between two locations has been added.
+
+
+### Web framework
+
+#### New and changed features
+
+- Web engine
+  - Feature support
+    - Support for OffscreenCanvas has been added: OffscreenCanvas enables canvas rendering apart from the main thread.
+  - Feature change/fix
+    - Strict MIME type for module scripts: Loading module script from the local file feature has been removed.
+    - App control URL bug fix: appcontrol:// scheme now able to launch application from encoded URL.
+- Web runtime
+  - Device API has been added for wrt-service.
+  - Web server and web server has been added as a wrt-service built-in module.
```

Comment:
Is the Web server repetition intentional or can we delete one Web server in the sentence?
```suggestion 

  - Web server has been added as a wrt-service built-in module.
```